HealthSDK/GiniHealthSDK/Sources/GiniHealthSDK/Core/GiniHealth.swift

```diff
@@ -57,7 +59,7 @@ public struct DataForReview {
  */
 @objc public final class GiniHealth: NSObject {
     /// reponsible for interaction with Gini Health backend .
-    public var giniApiLib: GiniHealthAPI
+    public var giniApiLib: HealthAPI
```

Comment:
[Clarification]:  Could you please double check I think we should keep here `GiniHealthAPI` 🤔

Reply:
I've added a protocol called HealthAPI for mocking it into UnitTests. GiniHealthAPI conforms to HealthAPI protocol. Do you have other renaming recommendation?